Fix 64 bits not being set by default

mikhailramalho opened the pull request:
Signed-off-by: Mikhail Gadelha <mikhail.ramalho@gmail.com>

mikhailramalho committed:
Fix 64 bits not being set by default

Signed-off-by: Mikhail Gadelha <mikhail.ramalho@gmail.com>

lucasccordeiro commented:
Some of the checks have not passed. 
@rafaelsamenezes: could you please check these problems?

Commit:
Added 32 bits option to tests

Signed-off-by: Mikhail Gadelha <mikhail.gadelha@sidia.com>

Commit:
Print the name of the failing test

Signed-off-by: Mikhail Gadelha <mikhail.gadelha@sidia.com>

Commit:
added fixedbv option to c2goto

Signed-off-by: Mikhail Gadelha <mikhail.gadelha@sidia.com>

Commit:
Removed define never defined

Signed-off-by: Mikhail Gadelha <mikhail.gadelha@sidia.com>

Commit:
Fix a bunch of errors when running with --floatbv and --ir

There were several checks to encode the program in integer
mode when using fixedbv but very few when using floatbv,
this commit just add the checks to floatbv encoding as well

Signed-off-by: Mikhail Gadelha <mikhail.gadelha@sidia.com>

Commit:
Fixed some test descriptions

Signed-off-by: Mikhail Gadelha <mikhail.gadelha@sidia.com>

Commit:
Removed digital filter models and tests

Signed-off-by: Mikhail Gadelha <mikhail.gadelha@sidia.com>

Commit:
Try to fix issue on the build server

Signed-off-by: Mikhail Gadelha <mikhail.gadelha@sidia.com>

Commit:
Added 32 bits option to kind tests

Signed-off-by: Mikhail Gadelha <mikhail.gadelha@sidia.com>

Commit:
The C++ frontend is broken, disable the regression

Signed-off-by: Mikhail Gadelha <mikhail.gadelha@sidia.com>

Commit:
Fix inconsistent handling of dynamic sizes

__ESBMC_alloc_size was an unsigned long (which has a different
size in 32 bits and 64 bits archs) and we were assigning various
objects with different types to it, in particular, int and pointer_type.

It was OK in 32 bits archs because these types have the same bit-width
but hell broke loose when we set 64 bits to be the default arch in
ESBMC.

Signed-off-by: Mikhail Gadelha <mikhail.gadelha@sidia.com>

Commit:
Removed duplicated cond

Signed-off-by: Mikhail Gadelha <mikhail.gadelha@sidia.com>
